FixedWing_Demo.py

```diff
@@ -0,0 +1,97 @@
+"""Spawn a single fixed wing UAV on x=0, y=0, z=50, with 0 rpy."""
```

Comment:
Don't need this file since there's one already under examples.

Reply:
If you really need it, [refer to this](https://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste)

Reply:
Will remove this file